imports/server/mediasoup.ts

```diff
@@ -335,12 +339,32 @@ class SFU {
 
     const observerAppData = observer.appData as AudioLevelObserverAppData;
 
-    const updateLastActivity = throttle(Meteor.bindEnvironment(() => {
-      CallHistories.upsert({
-        hunt: observerAppData.hunt,
-        call: observerAppData.call,
-      }, { $set: { lastActivity: new Date() } });
-    }), RECENT_ACTIVITY_TIME_WINDOW_MS);
+    const updateLastActivity = throttle(Meteor.bindEnvironment(
+      (volumes: types.AudioLevelObserverVolume[]) => {
+        CallHistories.upsert({
+          hunt: observerAppData.hunt,
+          call: observerAppData.call,
+        }, { $set: { lastActivity: new Date() } });
```

Comment:
I left our old call activity record around for the time being, but I suspect that we may be able to clean it up once all of the new activity stuff is implemented and displayed.